rspec-tools/rspec_tools/validation/description.py

```diff
@@ -16,8 +16,26 @@ def parse_names(path):
   section_names_path = read_file(path)
   return [s.replace('* ', '').strip() for s in section_names_path if s.strip()]
 
+def parse_security_standard_links(descr):
+  link_nodes = descr.find_all('a')
+  security_standards_links: Dict[str: List] = {}
+  for node in link_nodes:
+    href = node.attrs['href']
+    for standard, url_pattern in SECURITY_STANDARD_URL.items():
+      result = re.match(url_pattern, href)
+      if result is not None:
+        category = f"A{result[1].lstrip('0')}"
+        if standard not in security_standards_links.keys():
+          security_standards_links[standard] = []
+        security_standards_links[standard].append(category)
```

Comment:
The code here currently only works for the OWASP Top 10, and I know we wanted to extend this to CWE as well.

We can make this a but more generic by having each standard define a way to format the category.
```suggestion
        standard_id = result[1]
        if "convert_id" in standard_data and callable(standard_data["convert_id"]):
          standard_id = standard_data["convert_id"](standard_id)
        if standard not in security_standards_links.keys():
          security_standards_links[standard] = []
        security_standards_links[standard].append(standard_id)
```